src/main/java/org/broadinstitute/consent/http/resources/DataAccessRequestResource.java

```diff
@@ -182,11 +182,33 @@ public Response getInvalidDataAccessRequest() {
     @Path("/{id}")
     @Produces("application/json")
     @PermitAll
-    public Document describe(@PathParam("id") String id) {
-        return dataAccessRequestAPI.describeDataAccessRequestById(id);
+    public Response describe(@PathParam("id") String id) {
+        try {
+            new ObjectId(id);
+        } catch (IllegalArgumentException e) {
+            String message = "The provided id is not a valid Data Access Request Id: " + id;
+            logger.log(Level.INFO, message + "; " + e.getMessage());
+            return Response
+                    .status(Response.Status.BAD_REQUEST)
+                    .entity(new Error(message, Response.Status.BAD_REQUEST.getStatusCode()))
+                    .build();
+        }
+        try {
+            Document document = dataAccessRequestAPI.describeDataAccessRequestById(id);
+            if (document != null) {
+                return Response.status(Response.Status.OK).entity(document).build();
+            }
+            return Response
```

Comment:
```suggestion
          return createExceptionResponse(new NotFoundException("Unable to find Data Access Request with id: " + id));
```
Maybe it could be simplified like this

Reply:
I considered that - but I didn't what the logging that comes along with `createExceptionResponse` In these cases, a 400 or 404 is not really something we need to be logging at the warning level and sending to Sentry - it's somewhat expected that these kinds of things will happen and don't need any followup action. 

I think it might make sense to rename `createExceptionResponse` to something more in line with what it does ... log the error and try to decode the unexpected exception.